lgc/builder/InOutBuilder.cpp

```diff
@@ -1448,6 +1448,23 @@ Value *InOutBuilder::CreateTaskPayloadAtomicCompareSwap(AtomicOrdering ordering,
                          {getInt32(static_cast<unsigned>(ordering)), inputValue, comparatorValue, byteOffset}, {});
 }
 
+// =====================================================================================================================
+// Create debug printf operation, and write to the output debug buffer
+// @args : Printf variable parameters
+// @instName : Instance Name
+llvm::Value *lgc::InOutBuilder::CreateDebugPrintf(llvm::ArrayRef<llvm::Value *> args, const llvm::Twine &instName) {
```

Comment:
Please remove unnecessary namespace prefixes in .cpp files: you shouldn't need the `llvm::` and `lgc::` here.